utils/config.go

```diff
@@ -33,6 +33,7 @@ type ConfigStructure struct { // nolint: maligned
 	SwiftPublishRoots      map[string]SwiftPublishRoot      `json:"SwiftPublishEndpoints"`
 	AzurePublishRoots      map[string]AzurePublishRoot      `json:"AzurePublishEndpoints"`
 	AsyncAPI               bool                             `json:"AsyncAPI"`
+	DatabaseEtcd           string                           `json:"databaseEtcd"`
```

Comment:
I think you also need to add a default value to line ~100 below. See failing tests.